ios/Capacitor/Capacitor.xcodeproj/project.pbxproj

```diff
@@ -1116,7 +1118,7 @@
 				ENABLE_USER_SCRIPT_SANDBOXING = YES;
 				GCC_C_LANGUAGE_STANDARD = gnu17;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
Just curious, was there a specific reason this was previously set to `17.0`?

Reply:
It's from a "CodableTests" subproject, probably when created, Xcode set the version to 17 as the default and was not changed. Doesn't really affect the project or the users, but changed it for consistency.